client/apps/edit/components/content/sections/social_embed/controls.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import React, { Component } from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import SectionControls from '../../section_controls/index'
+import { onChangeSection, removeSection } from 'client/actions/editActions'
+
+export class SocialEmbedControls extends Component {
+  static propTypes = {
+    onChangeSectionAction: PropTypes.func.isRequired,
+    removeSectionAction: PropTypes.func.isRequired,
+    section: PropTypes.object.isRequired,
+    sectionIndex: PropTypes.number
+  }
+
+  componentWillUnmount = () => {
+    const {
+      removeSectionAction,
+      section,
+      sectionIndex
+    } = this.props
+
+    if (!section.url) {
+      removeSectionAction(sectionIndex)
+    }
+  }
+
+  render () {
+    const {
+      onChangeSectionAction,
+      section
+    } = this.props
+
+    return (
+      <div className='EmbedControls'>
```

Comment:
Reminder to `SocialEmbedControls` on these too. Be aware it will probably change styling